kernel/nativelib.ml

```diff
@@ -37,7 +36,7 @@ let ( / ) = Filename.concat
 (* We have to delay evaluation of include_dirs because coqlib cannot be guessed
 until flags have been properly initialized *)
 let include_dirs () =
-  [Filename.get_temp_dir_name (); coqlib () / "kernel"; coqlib () / "library"]
+  [Filename.get_temp_dir_name (); Envars.coqlib () / "kernel"; Envars.coqlib () / "library"]
```

Comment:
Why these changes?

Reply:
I remove all `open Envars` so I can grep for who's using them, as all users need work.

For example `nativelib` will have to use a less adhoc compilation model to support Dune properly.